getting_started/step_by_step/intro_to_the_editor_interface.rst

```diff
@@ -103,7 +103,7 @@ You can see four workspace buttons at the top: 2D, 3D, Script and
 AssetLib.
 
 You’ll use the **2D workspace** for all types of games. On top of 2D
-games that is where you’ll build your interfaces. Press F1 to access it.
+games, that is where you’ll build your interfaces. Press F1 to access it.
```

Comment:
Instead of just adding a comma can you instead change it to "In addition to 2D games, the 2D workspace is where you'll build your interfaces."

Reply:
Agreed.